app/settings/user-settings/user-settings.html

```diff
@@ -40,39 +40,66 @@ <h2 class="form-sheet-title" translate="settings.settings_list.user_settings">
             </div>
 
             <img class="field-logo" src="/img/hdx-logo.png" />
+            <form name="api">
+                <div
+                    class="form-field init required"
+                    ng-class="{'error': api.$invalid && api.$dirty,'success': !api.$invalid && api.$dirty}"
+                    ng-show="!hxlUserId"
+                >
+                
+                    <label class="required " for="hdx-user-id" translate="settings.user_settings.hdx_user_id">
+                        HDX User Id
+                    </label>
+                    <p translate="settings.user_settings.profile_page"> Found on your Humanitarian Data Exchange (HDX) profile page</p>
+                    <input id="hdx-user-id" name="hdx-user-id" ng-required="true" type="text" ng-model="hdxSettings['hdx_user_id'].config_value" required="required"/>
+                </div>
 
-            <div class="form-field alert error" ng-show="api.$invalid && api.$dirty && !hxlApiKey">
-                <svg class="iconic">
-                    <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/img/iconic-sprite.svg#warning"></use>
-                </svg>
-                <p translate="settings.user_settings.valid_key">A valid key is required</p>
-            </div>
+                <div class="form-field alert error" ng-show="api.$invalid && api.$dirty">
+                    <svg class="iconic">
+                        <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/img/iconic-sprite.svg#warning"></use>
+                    </svg>
+                    <p translate="settings.user_settings.valid_user">A user id is required</p>
+                </div>
 
-            <div
-                class="form-field init required"
-                ng-class="{'error': api.$invalid && api.$dirty,'success': !api.$invalid && api.$dirty}"
-                ng-show="!hxlApiKey"
-            >
-                <form name="api">
+                <div
+                    class="form-field init required"
+                    ng-class="{'error': api.$invalid && api.$dirty,'success': !api.$invalid && api.$dirty}"
+                    ng-show="!hxlApiKey"
```

Comment:
Same with this field - it really needs a meaningful name

Reply:
Yeah, there were the names it already had but I'll change them to explain they're bools.